floodmodeller_api/test/test_hydrographplus.py

```diff
@@ -0,0 +1,80 @@
+"""Tests to check the class HydrographPlus"""
+
+import os
+from pathlib import Path
+
+import pandas as pd
+import pytest
+
+from floodmodeller_api.hydrology_plus.hydrograph import HydrographPlusExport
+
+
+@pytest.fixture()
+def metada_csv():
```

Comment:
rename to something like `expected_metadata`, avoid calling it a csv as it is not a csv